sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/ng/SqlBase.g4

```diff
@@ -102,6 +102,15 @@ statement
         (PARTITIONED ON identifierList)?
         (TBLPROPERTIES tablePropertyList)? AS query                    #createView
     | ALTER VIEW tableIdentifier AS? query                             #alterViewQuery
+    | ALTER VIEW from=tableIdentifier AS? RENAME TO to=tableIdentifier #renameView
```

Comment:
Fine, I will combine them for `renameView`, `setViewProperties`, `unsetViewProperties`. 

I still want to keep `addViewPartition` and `dropViewPartitions`. These two have different syntax. Different from Alter Table, `addViewPartition` is unable to accept the `location` clauses; `dropViewPartitions` is unable to accept `PURGE`. 

I am wondering if that makes sense? Thanks!